Game/Source/GDKTestGyms/DeltaSerializeStruct.cpp

```diff
@@ -0,0 +1,63 @@
+#include "DeltaSerializeStruct.h"
+#include "Net/UnrealNetwork.h"
+
+void FTestStructItem::PostReplicatedChange(const struct FTestStructContainer& InArraySerializer)
```

Comment:
There are callbacks available for individual elements within a fas, couldn't that be used here instead?

Reply:
Which include the element idx, so you don't need to do the pointer magic